src/s3.rs

```diff
@@ -11137,22 +11148,25 @@ impl<P, D> S3Client<P, D> where P: ProvideAwsCredentials, D: DispatchSignedReque
     /// request parameters as selection criteria to return a subset of the objects in
     /// a bucket.
     pub fn list_objects(&self, input: &ListObjectsRequest) -> Result<ListObjectsOutput, S3Error> {
-        let mut request = SignedRequest::new("GET", "s3", self.region, "/{Bucket}");
+        let mut uri = String::from("/");
+        uri = uri +  &input.bucket.to_string();
```

Comment:
Isn't there a way to remove this concatenation with something like `format!("/{}", &input.bucket.to_string())`?